test/D2L.Security.OAuth2.WebApi.IntegrationTests/Authorization/NoImpersonationAttributeTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using D2L.Services;
+using NUnit.Framework;
+
+namespace D2L.Security.OAuth2.Authorization {
+	[TestFixture]
+	internal sealed class NoImpersonationAttributeTests {

```

Comment:
Not really worried about that; this attribute doesn't make sense without an `[Authentication]` (which will make that `null` check in the attribute not a concern) so not really interested in validating any particular behavior. If a route is anonymous there isn't any harm we could do either way.